templates/workspaces/base/porter.yaml

```diff
@@ -116,6 +116,12 @@ parameters:
     default: true
   - name: arm_environment
     type: string
+  - name: enable_cmk_encryption
+    type: boolean
+    default: false
+  - name: key_store_id
```

Comment:
I see the default is an empty string. If enable_cmk_encryption is true and the key is empty, will it work?

Reply:
No, it won’t work.
The default is an empty string only to make the key optional when `enable_cmk_encryption` is false.